cli/command/version/version.go

```diff
@@ -75,20 +75,18 @@ func showVersion(c cli.Interface) error {
 		},
 	}
 
-	conn, err := c.ClientConn()
-	if err == nil {
-		client := version.NewVersionClient(conn)
-		reply, err := client.Get(context.Background(), &version.GetRequest{})
-		if err != nil {
-			return fmt.Errorf("%s", grpc.ErrorDesc(err))
-		}
-		v.Server = &version.Info{
-			Version:   reply.Info.Version,
-			Build:     reply.Info.Build,
-			GoVersion: reply.Info.GoVersion,
-			Os:        reply.Info.Os,
-			Arch:      reply.Info.Arch,
-		}
+	conn := c.ClientConn()
```

Comment:
I pushed an update to get this expedited. Now we have `Connect()` and `ClientConn` methods. Who says you can't have your cake and eat it, too?

Not connected:
```
$ amp --server localhost:50000 version
grpc: addrConn.resetTransport failed to create client transport: [connection error: desc = "transport: dial tcp [::1]:50000: getsockopt: connection refused" {localhost:50000 <nil>}]; Reconnecting to %!v(MISSING)
grpc: addrConn.resetTransport failed to create client transport: [connection error: desc = "transport: dial tcp: operation was canceled" {localhost:50000 <nil>}]; Reconnecting to %!v(MISSING)Client:
 Version:       v0.6.0-dev
 Build:         48c183ac
 Server:        localhost:50000
 Go version:    go1.8
 OS/Arch:       darwin/amd64

Server:         not connected
```

Connected:
```
$ amp --server localhost:50101 version
Client:
 Version:       v0.6.0-dev
 Build:         48c183ac
 Server:        localhost:50101
 Go version:    go1.8
 OS/Arch:       darwin/amd64

Server:         
 Version:       v0.6.0-dev
 Build:         7f736a11
 Go version:    go1.8.1
 OS/Arch:       linux/amd64
```